web/cypress/integration/upstream/create_and_delete_upstream.spec.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create and Delete Upstream', () => {
+  const name = `upstreamName${new Date().valueOf()}`;
+  const sleepTime = 100; // the unit is milliseconds
+  const domSelectors = {
+    notification: '.ant-notification-notice-message'
+  };
+
+  beforeEach(() => {
+    // init login 
+    cy.login();
+  })
+
+  it('create upstream with default type (roundrobin)', () => {
+    // go to upstream create page
+    cy.visit('/');
+    cy.contains('Upstream').click();
+    cy.wait(sleepTime * 5);
+    cy.contains('Create').click();
+
+    // input name and description
+    cy.get('#name').type(name);
+    cy.get('#desc').type('desc_by_autotest');
+
+    // input information
+    cy.get('#nodes_0_host').type('127.0.0.1');
+    cy.get('#nodes_0_port').clear().type('7000');
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+    cy.get(domSelectors.notification).should('contain', 'Create upstream successfully');
```

Comment:
we could use a global file to contain those public text.

Reply:
agree, this will be carried out in another PR.